fast_and_reliable.py

```diff
@@ -0,0 +1,37 @@
+# C:\Users\IMOE001\Downloads\words.txt
+import re
+from time import process_time
+import typing
+
```

Comment:
Nice code :)
Your usage of type hinting, the fact that you seperated it into two different functions - it's great.